openvpn/factory.go

```diff
@@ -30,6 +30,7 @@ func NewServerConfig(
 	config.SetPingTimerRemote()
 	config.SetPersistTun()
 	config.SetPersistKey()
+	config.setFlag("explicit-exit-notify")
```

Comment:
Would be nice to have comment that problem it fixes (command, commit, anythere)

Reply:
Good point for the future